mregsite/urls.py

```diff
@@ -3,10 +3,12 @@
 
 from mreg.api.v1 import views
 
-from rest_framework_swagger.views import get_swagger_view
+from rest_framework.schemas import get_schema_view
+from rest_framework.permissions import AllowAny
 
 # Schema view for swagger api documentation
-schema_view = get_swagger_view(title='mreg API')
+schema_view = get_schema_view(title='mreg API',
+                              permission_classes=[AllowAny])
```

Comment:
Maybe add a test that the docs are readable without any authentication?

Reply:
@oyvindkolbu Thanks for reviewing.  I decided to drop the second commit entirely for now.  AIUI the `docs/` endpoint should provide a full API overview, but as of currently it only documents itself (i.e. `GET /docs`, not very useful).  That behaviour is the same before and after this PR.

I'll look into adjusting `/docs` to provide an actual OpenAPI UI in a separate PR.